virtcontainers/container.go

```diff
@@ -1310,10 +1310,21 @@ func (c *Container) updateMemoryResources(oldResources ContainerResources, newRe
 	}).Debug("Request update memory")
 
 	if oldMemMB == newMemMB {
-		c.Logger().WithFields(logrus.Fields{
-			"old-mem": fmt.Sprintf("%dMB", oldMemMB),
-			"new-mem": fmt.Sprintf("%dMB", newMemMB),
-		}).Debug("the actual number of Mem will not be modified")
+		c.Logger().Debug("the current memory will not be modified")
+		addMemDevice := &memoryDevice{
+			sizeMB: 0,
+		}
+		data, err := c.sandbox.hypervisor.hotplugAddDevice(addMemDevice, memoryDev)
```

Comment:
Why are we trying to hotplug more memory here? We're in the case where `oldMemMB == newMemMB`, and as mentioned by the debug log `the current memory will not be modified`.